staff/aleix-caminal/cv/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <title>Curriculum Vitae</title>
+    </head>
+    <body>
+        <div class="container">
+            <section id="head">
+                <div class="image">
+                    <img src="assets/img/profile.jpg" alt="profile">
+                </div>
+                <div class="info">
+                    <h4 class="name">Charles Tucker III</h4>
+                    <p class="web">Web: <a href="http://memory-alpha.wikia.com/wiki/Charles_Tucker_III">http://memory-alpha.wikia.com/wiki/Charles_Tucker_III</a></p>
+                    <p class="email">E-mail: <a href="mailto:charlestrip@starfleet.com">charlestrip@starfleet.com</a></p>
+                    <p class="mobile">Mobile: 07791 858292</p>
+                    <address>
+                        <p class="street">8 Louisiana Ave</p>
+                        <p class="town">Panama City, Florida</p>
+                        <p class="country">North America, United Earth</p>
+                    </address>
+                </div>
```

Comment:
Forbiden use DIV :)

Reply:
is not forbidden, anyways i've changed it